src/middlewared/middlewared/plugins/replication.py

```diff
@@ -52,14 +52,18 @@ class Config:
         progressfile = f'/tmp/.repl_progress_{data["id"]}'
         if os.path.exists(progressfile):
             with open(progressfile, 'r') as f:
-                pid = int(f.read())
-            title = await self.middleware.call('notifier.get_proc_title', pid)
-            if title:
-                reg = re.search(r'sending (\S+) \((\d+)%', title)
-                if reg:
-                    data['status'] = f'Sending {reg.groups()[0]}s {reg.groups()[1]}s'
-                else:
-                    data['status'] = 'Sending'
+                f.seek(0)
```

Comment:
What do you intend to do?

Reply:
@themylogin sorry, do you mean with the `f.seek(0)` call? I put that in there to ensure we start at the beginning of the file before reading it's contents. Customer received a traceback in the alert system because the replication progress file was empty.

Reply:
But files are always opened at the beginning?

Reply:
@themylogin just a habit I have I guess. If f.read() is called twice in the same context, it will return EOF. The traceback showed an empty string and I couldn't definitively say if it was because it was empty. I'll go ahead and remove it.